src/main/java/gui/MealPlanGeneratorGUI.java

```diff
@@ -183,9 +188,9 @@ public MealPlanGeneratorGUI(String userType) {
             vegetarianCheckBox.setText(MealPlanConstants.VEG);
             numFoodsPrompt.setText(MealPlanConstants.NUM_FOODS);
             invalidInput.setText(Exceptions.INVALID_INPUT);
-            enterButton.setText("Enter");
+            enterButton.setText("ENTER");
             intro.setText(MealPlanConstants.MEALPLAN_INTRO_GUI);
-            returnToMenu.setText("Return to Menu");
+            returnToMenu.setText("RETURN TO MENU");
```

Comment:
maybe we should factor out these "enter" and "return to Menu" into the constants as many of these are used throughout?